packages/react-events/src/dom/shared/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+import {DiscreteEvent, UserBlockingEvent} from 'shared/ReactTypes';
+
+export const hasPointerEvents =
```

Comment:
At some point we'll need to avoid inlining common internal dependencies in the builds for each responder.